index.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @defgroup plugins_generic_backup Backup Plugin
+ */
+ 
+/**
+ * @file plugins/generic/backup/index.php
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @brief Wrapper for backup plugin.
+ *
+ * @ingroup plugins_generic_backup
+ */
+
+require_once('BackupPlugin.inc.php');
+
+return new BackupPlugin();
+
+?> 
```

Comment:
(Good practice to remove these ending PHP tags when you get the chance)

Reply:
removed.